app/views/ui-components/v1/forms/broker_registration/_broker_agency_information.html.slim

```diff
@@ -1,4 +1,4 @@
-h4.mb-2 NOTE: If you are not associated with a broker agency, please enter your personal information. Do not enter a General Agency or Third Party Administrator‘s Information.
+h4.mb-2 NOTE: If you are not associated with a #{l10n("broker_agency")}, please enter your personal information. Do not enter a General Agency or Third Party Administrator‘s Information.
```

Comment:
This is going to read "a expert."

Reply:
I'll change the whole thing to a translation